static/src/javascripts/projects/common/modules/ui/subscription-banners/subscription-banner.js

```diff
@@ -181,7 +181,7 @@ const createBannerShow = (
 };
 
 const chooseBanner = (region: ReaderRevenueRegion) =>
-    region === 'australia' ? gwBannerTemplate : subscripionBannerTemplate;
+    (region === 'australia' || region === 'rest-of-world') ? gwBannerTemplate : subscripionBannerTemplate;
```

Comment:
I notice there's a typo in `subscripionBannerTemplate` I know it was not introduced in this PR but it would be good if you could fix it.

Reply:
Done 👍